### PR TITLE
Add additional qc metrics

### PR DESCRIPTION
**Pull request by susannasiebert**
See Jira issue https://jira.gsc.wustl.edu/browse/APIPE-3242

### CODE REVIEW

File: lib/perl/Genome/Qc/Config.pm [hunk 1]
```diff
@@ -45,6 +45,12 @@ sub get_commands_for_alignment_result {
                 use_version => 1.123,
             },
         },
+        samtool_flagstat => {
```
Comment (tmooney) [inline lib/perl/Genome/Qc/Config.pm]: Should this be `samtools_flagstat`?

Comment (susannasiebert) [inline lib/perl/Genome/Qc/Config.pm]: Yes, it's a typo (although this hash key doesn't have to match the tool; it can really be anything). It's now fixed.

**Comment by tmooney** [comment]
+1, assuming that it would be silly for this to ever be run on something with 0 `total_reads`.

**Comment by mkiwala** [comment]
+1

**Pull request merged**